samples/batch-synthesis/csharp/BatchSynthesisSample/Program.cs

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+
+// Your Speech resource key and region
+// This example requires environment variables named "SPEECH_KEY" and "SPEECH_REGION"
+string speechRegion;
+string speechKey;
+
+if (Environment.GetEnvironmentVariable("SPEECH_REGION") is string regionValue)
+{
+    speechRegion = regionValue;
+}
+else
+{
+    throw new ArgumentException($"Please set the SPEECH_REGION environment variable to set speech resource region.");
+}
+
+if (Environment.GetEnvironmentVariable("SPEECH_KEY") is string keyValue)
+{
+    speechKey = keyValue;
+}
+else
+{
+    throw new ArgumentException($"Please set the SPEECH_KEY environment variable to set speech resource key.");
+}
+
+var host = $"https://{speechRegion}.customvoice.api.speech.microsoft.com";
+
+var sampleScript = "This is the waistline, and it's falling.\nAlexis, meet Bill and Hillary, and the rest of America.\nThis is Jordan, Scottie Pippen and the ring dynasty.\nThe more I looked, the gloomier I got.\nHe saw Macbeth, the three witches, and the boiling cauldron.\nBut it is good to get together, cook together, eat together and enjoy.\nThe priorities, they say, are sofa, blanket, and clothing.\nThe metallic walls curve, twist, and turn.\nThe seagrass fiber is tough, durable and smooth.\nThe dissenters were Stevens, Souter, Ginsburg and Breyer.";
```

Comment:
Let's use Gatsby-chapter1.txt as the Python sample does.

Reply:
good point. let me update